src/_tests_/components/forecast-summary.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import ForecastSummary from "../../components/forecast-summary";
+
+afterEach(cleanup);
```

Comment:
don't need this anymore, it's not breaking anything, just unnecessary code nowadays